programmers/난이도별/level01.예산/dkdlelk99.py

```diff
@@ -0,0 +1,13 @@
+def solution(d, budget):
+    answer = 0
+    d.sort()
+    for i in d:
+        budget -= i
+        answer += 1
+        if budget < 0:
+            answer -= 1
+            break
```

Comment:
```suggestion
        if budget < 0: break
        answer += 1
```
이렇게 하는게 더 자연스러울 것 같아요!